### DISCUSSION TITLE
Software´s shereable

### DISCUSSION THREAD

**Discussion by CAVERNASTUDIO** [audacity/audacity#1954] [Q&A]
Is there any software that can work as a sequencer in audacity?, I can create a loop, no problem. Performing sequences with midi software is it possible?

**Comment by SteveDaulton** [comment]
Audacity's MIDI abilities are very limited (Audacity is primarily an app for recording / editing / processing "audio")
Perhaps worth taking a look at MuseScore https://musescore.org/en or do an Internet search for open source MIDI sequencer.
